x-pack/plugins/case/server/connectors/index.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Logger } from 'kibana/server';
+import {
+  ActionTypeConfig,
+  ActionTypeSecrets,
+  ActionTypeParams,
+  ActionType,
+  // eslint-disable-next-line @kbn/eslint/no-restricted-paths
+} from '../../../actions/server/types';
+import {
+  CaseServiceSetup,
+  CaseConfigureServiceSetup,
+  CaseUserActionServiceSetup,
+} from '../services';
+
+import { getActionType as getCaseConnector } from './case';
+
+export interface GetActionTypeParams {
+  logger: Logger;
+  caseService: CaseServiceSetup;
+  caseConfigureService: CaseConfigureServiceSetup;
+  userActionService: CaseUserActionServiceSetup;
+}
+
+export interface RegisterConnectorsArgs extends GetActionTypeParams {
+  actionsRegisterType<
+    Config extends ActionTypeConfig = ActionTypeConfig,
+    Secrets extends ActionTypeSecrets = ActionTypeSecrets,
+    Params extends ActionTypeParams = ActionTypeParams,
```

Comment:
why the `=`? Does just this work:
```suggestion
    Config extends ActionTypeConfig,
    Secrets extends ActionTypeSecrets,
    Params extends ActionTypeParams,
```

Reply:
I am not sure. I wanted to match the action's definition. https://github.com/elastic/kibana/blob/master/x-pack/plugins/actions/server/plugin.ts#L86